### PR TITLE
Launch CFP

### PR DESCRIPTION
**Pull request by roidelapluie**
Signed-off-by: Julien Pivotto <roidelapluie@inuits.eu>

### CODE REVIEW

File: content/2021-losangeles/submit.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+[CNCFcolocatedevents@cncf.io](mailto:CNCFcolocatedevents@cncf.io).
+
+
+<a class="btn btn-default btn-lg" href="https://linuxfoundation.smapply.io/prog/promcon_north_america_2021/" target="_blank" role="button">
+  <i class="fa fa-commenting"></i>&nbsp;&nbsp;Submit Talk
+</a>
+
+## Dates to remember
+
+- CFP opens: Monday, June 28
```
Comment (juliusv) [inline content/2021-losangeles/submit.md:20]: Title case the "opens" -> "Opens" like the other bullet points or do it all the other way around?

File: content/2021-losangeles/submit.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+- CFP Closes: Sunday, July 25, 11:59pm PDT
+- CFP Notifications: Friday, August 20
```
Comment (juliusv) [inline content/2021-losangeles/submit.md:22]: Could be clearer to say "Talk Acceptance Notifications" or something like that?

File: lib/helpers/conferences.rb [hunk 3]
```diff
@@ -108,6 +108,7 @@ def conferences
       {
         '/2021-losangeles/' => 'Overview',
         '/2021-losangeles/registration/' => 'Registration',
+        '/2021-losangeles/submit/' => 'CfP',
```
Comment (juliusv) [inline lib/helpers/conferences.rb:111]: In the past we've called this "Submit a talk". Is the motivation for "CfP" the menu space issue? Anyway, fine for now, but I'd prefer to not rely on people having to know the abbreviation.

Comment (roidelapluie) [reply lib/helpers/conferences.rb:111]: This is a copy and paste from last year

**Comment by roidelapluie** [comment]
Thank you @juliusv, I appreciate the review 💯

File: content/2021-losangeles/submit.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+
+<a class="btn btn-default btn-lg" href="https://linuxfoundation.smapply.io/prog/promcon_north_america_2021/" target="_blank" role="button">
+  <i class="fa fa-commenting"></i>&nbsp;&nbsp;Submit Talk
+</a>
+
+## Dates to remember
+
+- CFP opens: Monday, June 28
+- CFP Closes: Sunday, July 25, 11:59pm PDT
```
Comment (juliusv) [inline content/2021-losangeles/submit.md:21]: This one is still "Closes" and not "closes"

**Comment by juliusv** [comment]
:+1: Besides last nit

**Pull request merged**